### DISCUSSION TITLE
Fuuusão #2

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#444] [Imagens]
Post de referência: https://www.vish.blog.br/imagens/fuuusao-2/

**Comment by angellisy** [comment]
Dá pra ver que é uma simples sobreposição de camadas, mas ainda assim, a capa ficou incrível

**Comment by vish-blog** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/imagens/fuuusao-2/#comment_1173951910>):

> Dá pra ver que é uma simples sobreposição de camadas, mas ainda assim, a capa ficou incrível

:sunglasses:
